examples/publisher-reactjs.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import Axios from 'axios'
+import { connect } from 'react-redux'
+import querystring from 'querystring'
+import store from '../../store'
+
+class MercureTest extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
```

Comment:
Shouldn't these values be stored in a `const` defined before the class instead of in a state? Also, couldn't we use a functional component instead of a class?

Reply:
Sure, sorry, I'm new in React and this component is an extract from a component I use in my project (and more complicated than here).

Here, I need componentDidMount function so I don't really see how to make a function insteed of a component with it.